tests/integration/modules/molecule/pipe_handler/playbook.yml

```diff
@@ -0,0 +1,140 @@
+---
+- name: Converge
+  collections:
+    - sensu.sensu_go
+  hosts: all
+  gather_facts: true
```

Comment:
You probably copy-pasted this, but we could actually set this to `no` because we're just accessing the REST API, and gathering facts is time consuming. Let me open issue that we must fix it for all modules; integration tests will run quite faster I think

Reply:
UPDATE: actually, leave it for now, we can do a followup, I opened an issue for it